vnext/ReactUWP/Views/PopupViewManager.cpp

```diff
@@ -62,13 +64,15 @@ void PopupShadowNode::createView() {
   m_popupClosedRevoker =
       popup.Closed(winrt::auto_revoke, [=](auto &&, auto &&) {
         auto instance = wkinstance.lock();
+        m_isWindowed = false;
         if (!m_updating && instance != nullptr)
           OnPopupClosed(*instance, m_tag, false);
       });
 
   m_popupOpenedRevoker =
       popup.Opened(winrt::auto_revoke, [=](auto &&, auto &&) {
         auto instance = wkinstance.lock();
+        m_isWindowed = true;
```

Comment:
Does it actually matter to track isWindowed or if m_isWindowed was always true things would still work?  What does it mean to measure something in a popup that is not open that it would behave differently

Reply:
if it can always be true a method override might be better than a new member on every node

Reply:
At one point I was using this flag for dual purposes ; 1) to designate windowed XAML elements, and 2) to indicate that the element was open. Now, though, I don't think it's possible to get to the NativeUIManager::measure without the window being opened. I'll verify that and rework that piece

---
In reply to: [305402848](https://github.com/microsoft/react-native-windows/pull/2781#discussion_r305402848) [](ancestors = 305402848)

Reply:
I updated the PR with new commits.